lib/flutter_command.dart

```diff
@@ -103,7 +103,8 @@ class CommandError<TParam> {
 /// where [TParam] is the type of data that is passed when calling [execute] and
 /// [TResult] denotes the return type of the handler function. To signal that
 /// a handler doesn't take a parameter or returns no value use the type `void`
-abstract class Command<TParam, TResult> extends ValueNotifier<TResult> {
+abstract class Command<TParam, TResult>
+    extends _ValueEqualityNotifier<TResult> {
```

Comment:
I added a `CustomValueNotifier` that does the same thing and its already used for thrownExceptions. Please use thaat one

Reply:
Sorry I didn't get your hint here. 😃 
The one used for `thrownExceptions` is to count the listeners right. But this functionality is not required for the Command I assume. Hence I went with the idea of having a separate `_ValueEqualityNotifier`. I will look into this further in new NullSafe version. 👍

Reply:
Nope, the counting one is the `_commandResult` see
![grafik](https://user-images.githubusercontent.com/16635729/110105764-14fcf800-7da9-11eb-8fb5-660ee1881ac8.png)

Reply:
Ok this is in the updated version. I just pulled the repo to verify where `CustomValueNotifier` comes from. So its from `functional_listener`. I think it shouldn't be difficult to use that. Will work on this next week. 👍